docs/markdown/Style-guide.md

```diff
@@ -0,0 +1,36 @@
+---
+short-description: Style recommendations for Meson files
+...
+
+# Style recommendations
+
+This page lists some recommendations on organizing and formatting your
+Meson build files.
+
+## Tabs or spaces?
+
+Always spaces.
```

Comment:
Should we recommend 2 spaces as well?

Reply:
I specifically left that out, because I'm a bit torn on whether we should do 2 or 4.